apps/shelter-web/project.json

```diff
@@ -11,6 +11,18 @@
       "options": {
         "jestConfig": "apps/shelter-web/jest.config.ts"
       }
+    },
+    "build": {
+      "executor": "@nx/vite:build",
+      "options": {
+        "outputPath": "dist/apps/shelter-web",
+        "configFile": "apps/shelter-web/vite.config.mts"
+      },
+      "configurations": {
+        "production": {
+          "mode": "production"
+        }
+      }
```

Comment:
@vecchp FYI - I added this so the build command would respect the .env files. 
otherwise `nx build shelter-web --configuration=production` only loaded .env and ignored `.env.production`